backend/svirt.pm

```diff
@@ -21,13 +21,19 @@ use testapi qw(get_var get_required_var check_var);
 
 use IO::Select;
 
+use Class::Accessor "antlers";
+has instance => (is => "rw", isa => "Num");
+has name     => (is => "rw", isa => "Str");
+
```

Comment:
I would really prefer if the backend didn't duplicate this and just get the name from the console. Because you can change the name in the console (attach_to_running) and the backend wouldn't follow